src/benchmarks/micro/libraries/System.Text.RegularExpressions/Perf.Regex.Common.cs

```diff
@@ -38,14 +38,19 @@ public void Setup()
         }
 
         [Benchmark] public void Backtracking() => _notOneLoopNodeBacktracking.Match("Essential services are provided by regular exprs.");
-        [Benchmark] public void OneNodeBacktracking() => _oneNodeBacktracking.Match("This regex has the potential to be optimized further");
-        [Benchmark] public void Email_IsMatch() => _email.IsMatch("yay.performance@dot.net");
-        [Benchmark] public void Email_IsNotMatch() => _email.IsMatch("yay.performance@dot.net#");
+        [Benchmark] [MemoryRandomization]
+public void OneNodeBacktracking() => _oneNodeBacktracking.Match("This regex has the potential to be optimized further");
+        [Benchmark] [MemoryRandomization]
+public void Email_IsMatch() => _email.IsMatch("yay.performance@dot.net");
+        [Benchmark] [MemoryRandomization]
+public void Email_IsNotMatch() => _email.IsMatch("yay.performance@dot.net#");
 
-        [Benchmark] public void Date_IsMatch() => _date.IsMatch("Today is 11/18/2019");
+        [Benchmark]         [MemoryRandomization]
```

Comment:
Extra space

Reply:
I am going to reformat these so they are not all on the same line. It will look cleaner, and match up with what we have everywhere else.